co-log-benchmark/co-log-benchmark.cabal

```diff
@@ -0,0 +1,32 @@
+name:                co-log-benchmark
```

Comment:
Can you please update `cabal.project` file and add name of this package there?
And also update `.travis.yml` file for `stack` section to build this package on CI as well

Reply:
Yep, will do